src/components/Error/ErrorMessage.tsx

```diff
@@ -27,10 +29,12 @@ export default function ErrorMessage({ label, errorMessage }: Props) {
   }
 
   return (
-    <div className="ErrorMessage__Container">
-      <div className="ErrorMessage__Header">
-        <ErrorNotice />
-        <span>{label}</span>
+    <div className={classNames(className, 'ErrorMessage__Container')}>
```

Comment:
Maybe className has to be after `ErrorMessage__Container` if we want to override something without `!important` here?